### DISCUSSION TITLE
Length of games, when does the end game start?

### DISCUSSION THREAD

**Discussion by Ramza13** [CleverRaven/Cataclysm-DDA#48502] [Q&A]
I've been working on some ideas that like #48244 will scale over time.  In part this is to make the game more dynamic and to add more end game content that can be hard without destroying new characters.

One problem I am having is the time periods to scale things based off of.  The linked PR uses 0-14 days, 14-50 days and 50+ days as categories of time each of which behaves differently.  I made those numbers up randomly and would love some better numbers.  I've never gotten as far as some people into the 'end game' and would love your thoughts on when this tends to happen?

How long roughly do your successful characters live and when is it appropriate to start scaling up the difficulty?

**Comment by actual-nh** [comment]
~~You may want to take a look at the monster group start times - the elite survivor zombies start after 60 days, for instance.~~ Never mind, by <https://cataclysmdda.org/design-doc/> none of the start times are anywhere near even getting out of the early phase.

**Comment by Venera3** [comment]
In my experience the "win more" part of the game starts at around 2-3 months in. By that you most survivors will have  looted a military location for guns, have some CBMs/mutations and crucially have trivialized maintaining their vehicle. Car-fu and the mountain of 5,56 they are sitting on by this point make open-world threats a non-issue, and the car means every nontrivial encounter is optional.

For my critters I used 1/2/3 months as midgame(probably too late, but I don't want to penallize suboptimal play too much) / late midgame / lategame break points.

**Reply by actual-nh** [reply]
In other words, the monster group start times are likely to be a better guide currently than <https://cataclysmdda.org/design-doc/#game-stages-and-timeline> (which is more "aspirational" - what they eventually should be).

**Reply by actual-nh** [reply]
Note: Check the last comment in #47960 for something that may confuse things; that 60 days I mentioned - for elite zombie survivors - actually winds up being 240 (which may fit the design-doc timeline better) with default settings.

**Comment by robotic-gamer** [comment]
a few theoretical notes:
- the end game is subjective it can be getting survivor armor, being a post threshold mutant, building a npc base, having a full suit of power armor etc.
- also "end game" is an over simplification, really there are multiple end games/progression for every type of thing from armor, to weapons, mutagen production, base defense, vehicle transport, etc.
- a better question would be to find/figure out how players could/would interact with the specific thing you are considering adding.
- then list the interactions from early game to late game
-  you can then try to shift it to occur at the point player survival is probable or would be balanced.
- you could also skim YouTube  cdda playthroughs to get a idea when people generally get what.

**Comment by hexagonrecursion** [comment]
C:DDA is open-world and play styles vary. It would not surprise me if the characters of some players are more overpowered after the first two weeks than the characters of other players are after a year.
